episodes/superspreading-simulate.Rmd

```diff
@@ -0,0 +1,847 @@
+
+```{r,warning=FALSE,message=FALSE}
+library(outbreaks)
+
+mers_cumcases <- mers_korea_2015$linelist %>%
+  # incidence2 workflow
+  incidence2::incidence(date_index = "dt_onset") %>%
+  incidence2::complete_dates() %>%
+  # wrangling using {dplyr}
+  mutate(count_cumsum = cumsum(count)) %>%
+  rownames_to_column(var = "day") %>%
+  mutate(day = as.numeric(day))
+```
+
+Use `plot()` to make an incidence plot:
+
+```{r}
+# plot the incidence2 object
+plot(mers_cumcases)
+```
+
+:::::::::::::::::::::::::::::::::::
+
+When plotting the observed number of cumulative cases from the Middle East respiratory syndrome (MERS) outbreak in South Korea in 2015 alongside the previously simulated chains, we identify that the observed cases followed a trajectory that resembled a simulated explosive outbreak.
+
+```{r,echo=FALSE}
+# Observed cases vs Simulated transmission chains
+mers_cumcases_type <-
+  mers_cumcases %>%
+  mutate(type = "Observed")
+simulated_chains_day_type <-
+  simulated_chains_day %>%
+  mutate(type = "Simulated")
+
+ggplot() +
+  geom_line(
+    data = simulated_chains_day_type,
+    mapping = aes(
+      x = day,
+      y = cases_cumsum,
+      group = chain_id,
+      color = type
+    )
+  ) +
+  geom_hline(aes(yintercept = 100), lty = 2) +
+  geom_line(
+    data = mers_cumcases_type,
+    mapping = aes(
+      x = day,
+      y = count_cumsum,
+      color = type
+    ),
+    linewidth = 1.5
+  ) +
+  labs(
+    x = "Day since first report",
+    y = "Cumulative cases",
+    color = "Type"
+  )
+```
+
+When we increase the dispersion parameter from $k = 0.01$ to $k = \infty$, given a fixed reproduction number $R = 1.5$, the proportion of simulated outbreaks that reached the 100-case threshold increases.
+
+![**Growth of simulated outbreaks with R = 1.5 and one initial case, conditional on non-extinction.** Boxes show the median and interquartile range (IQR) of the first disease generation with 100 cases; whiskers show the most extreme values within 1.5 × IQR of the boxes, and crosses show outliers. Percentages show the proportion of 10,000 simulated outbreaks that reached the 100-case threshold ([Lloyd-Smith et al., 2005](https://www.nature.com/articles/nature04153)).](fig/see-nature04153_Fig2-c.jpg)
+
+:::::::::::::::::::::: testimonial
+
+### Early spread projections
+
+In the epidemic's initial phase, you can use `{epichains}` to apply a branching process model to project the number of future cases. Validation analysis showed that this is a rapid and low-detail model with forecast prediction intervals that are too certain ([Pearson et al., 2020](https://www.eurosurveillance.org/content/10.2807/1560-7917.ES.2020.25.18.2000543#validationofthemodel-1)).
```

Comment:
```suggestion
In the epidemic's initial phase, you can use `{epichains}` to apply a branching process model to project the number of future cases. Even though the model accounts for randomness in transmission and variation in the number of secondary cases, there may be additional local features we have not considered. Analysis of early forecasts made for COVID in different countries using this model structure found that predictions were often overconfident ([Pearson et al., 2020](https://www.eurosurveillance.org/content/10.2807/1560-7917.ES.2020.25.18.2000543#validationofthemodel-1)). This is likely because the real-time model did not include all the changes in the offspring distribution that were happening at the local level as a result of behaviour change and control measures. You can read more about the importance of local context in COVID-19 models in [Eggo et al. (2020)](https://www.nature.com/articles/s43588-020-00014-7).
```
Have tried to broaden implications a bit for users.